oarepo_ui/theme/assets/semantic-ui/js/oarepo_ui/search/FoldableBucketAggregationElement.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from "react";
+import { Accordion, Header, Card, Icon, Transition } from "semantic-ui-react";
+import PropTypes from "prop-types";
+
+export const FoldableBucketAggregationElement = ({ title, containerCmp }) => {
+  const [isActive, setIsActive] = useState(false);
+
+  const handleClick = () => setIsActive((prevState) => !prevState);
+  return (
+    <Card className="borderless facet foldable rel-ml-1">
+      <Accordion>
+        <Accordion.Title
+          onKeyDown={(e) => {
+            if (e.key === "Enter" || e.key === " ") {
+              handleClick();
+            }
+          }}
+          tabIndex={0}
+          active={isActive}
+          onClick={handleClick}
+        >
+          <div className="flex justify-space-between align-items-center">
+            <Header className="mb-0" as="h3">
+              {title}
+            </Header>
+            <div className="align-self-end">
+              <Icon name="angle right" />
```

Comment:
Does it rotate when activated ?

Reply:
@mirekys Yes, we already have css for it here https://github.com/oarepo/oarepo-ui/blob/eed9fa5abe831871ab461543cd29fb05fa3ea230/oarepo_ui/theme/assets/semantic-ui/less/oarepo_ui/themes/default/modules/accordion.overrides#L4 it uses the same principle that we already have in the accordions that are part of the deposit form. I would strongly recommend actually testing out this PR by yourself please, because it would be good to be sure that things are acting exactly like we discussed and that I did not misunderstand anything.

Reply:
Thanks, yes for sure, for the first round I was just quickly going through code on the subway :D